requirements.txt

```diff
@@ -1,5 +1,4 @@
 Cython==0.23.2
-numpy==1.9.2
-scikit-learn==0.16.1
-scipy==0.16.0
-wsgiref==0.1.2
+numpy==1.11.2
```

Comment:
Why the package upgrades?

Maybe instead of `==` they should be `>=`?

Reply:
travis build was failing from version mismatch between the scipy and numpy versions being used. these are just more up to date versions that i know experimentally are compatible.  I will change to >= at your request, I really don't care.